schema/patches/12.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE schedule.demo
+    ADD COLUMN signup_cutoff_hours INT DEFAULT 0;
```

Comment:
It would be nice if we could group sql patches together by topic. Could we remove 6.sql and merge it into 12.sql?

Reply:
The idea behind patches was that we never go back on them - each patch is an "upgrade" of the previous version, otherwise we could have fun stuff happen in production. In theory we could remove older patches when they're superseded, however that makes my head hurt more than just keeping them around forever.